src/Lib/App.hs

```diff
@@ -7,24 +7,40 @@ module Lib.App
        ) where
 
 import Control.Monad.Except (MonadError, throwError)
-import Control.Monad.Logger (LoggingT, MonadLogger, runStdoutLoggingT)
+import Katip (ColorStrategy (ColorIfTerminal), Katip, KatipContext, KatipContextT,
+              Severity (DebugS), Verbosity (V2), closeScribes, defaultScribeSettings, initLogEnv,
+              mkHandleScribe, registerScribe, runKatipContextT)
 import Servant.Server (Handler, err400, err401, err404, err500, errBody)
 
 import Lib.App.Env
 import Lib.App.Error (AppError (..))
 import Lib.Effects.Session (MonadSession)
 import Lib.Effects.User (MonadUser)
 
-newtype App a = App {
-    unApp :: LoggingT(ReaderT AppEnv (ExceptT AppError IO)) a
-} deriving (Monad, Functor, Applicative, MonadReader AppEnv, MonadError AppError, MonadIO, MonadLogger)
+-- TODO: inject logger configuration directly
+newtype App a = App
+  { unApp :: KatipContextT (ReaderT AppEnv (ExceptT AppError IO)) a
+  } deriving (Monad, Functor, Applicative, MonadReader AppEnv, MonadError AppError,
+              MonadIO, Katip, KatipContext)
 
 instance MonadSession App
 instance MonadUser App
 
 runAppAsHandler :: AppEnv -> App a -> Handler a
 runAppAsHandler env action = do
-  res <- liftIO $ runExceptT $ runReaderT (runStdoutLoggingT $ unApp action) env
+  -- TODO: make severity configurable
+  handleScribe <- liftIO $ mkHandleScribe ColorIfTerminal stdout DebugS V2
```

Comment:
How are we planning to make logger configurations?

Reply:
Yeah, I think at some point in future we will have such config.